contrib/golang/filters/http/source/go/pkg/plugins/echo/config.go

```diff
@@ -0,0 +1,15 @@
+package echo
```

Comment:
This should be moved to somewhere in the test hierarchy and used for a full integration test.

Reply:
Done